scipy/linalg/tests/test_basic.py

```diff
@@ -618,6 +618,20 @@ def test_random_complex(self):
             b = random([n, 3])
             x = solve(a, b)
             assert_array_almost_equal(dot(a, x), b)
+    
+    def test_sym_pos_dep(self):
+        n = 20
```

Comment:
we don't need to construct a 20x20 array here `solve([[1.]], [1], sym_pos=True` should be enough to generate the warning.